lib/jit_preloader/active_record/base.rb

```diff
@@ -48,6 +51,15 @@ def reload(*args)
       end
 
     end
+
+  end
+
+  def self.generate_key(method_name, conditions)
```

Comment:
this all being said, I think there is a nicer way to do this that doesn't even require this method :)


`key = "#{method_name}|#{conditions.sort.hash}"`